src/env.cr

```diff
@@ -109,10 +109,37 @@ module ENV
     end
   end
 
+  # Removes all environment variables.
   def self.clear : Nil
     keys.each { |k| delete k }
   end
 
+  # Conditionally merges *other* hash in, and uses the block to determine the merge value.
+  # The block is yielded the name of each duplicate entry, the initial-value and other-value, and expects the result of the block to be a String or Nil.
+  # If the block returns nil, the new entry is not merged into the ENV.
```

Comment:
I think this is better understandable:
```suggestion
  # If the block returns nil, the key is removed from the ENV.
```

Reply:
Updated to use this wording which is more clear.